### PR TITLE
Rename function parameters to be consistent.

### PR DESCRIPTION
**Pull request by nathanielc**
Renamed col -> column
Renamed table -> tables

### CODE REVIEW

File: docs/SPEC.md [hunk 1]
```diff
@@ -1413,8 +1413,8 @@ from(bucket:"telegraf/autogen")
 
 Median is defined as:
 
-    median = (method="estimate_tdigest", compression=0.0, table=<-) =>
-    	table
+    median = (method="estimate_tdigest", compression=0.0, tables=<-) =>
+    	tabels
```
Comment (ebb-tide) [inline docs/SPEC.md]: typo @ "tabels" !

Comment (nathanielc) [reply docs/SPEC.md]: thanks!

Comment (nathanielc) [reply docs/SPEC.md]: I just fixed them all, thanks!

**Comment by nathanielc** [comment]
PLatform PR here https://github.com/influxdata/platform/pull/1303 idpe didn't need one.

**Pull request merged**